### ISSUE TITLE
[Feature] Add Fling support

### ISSUE THREAD

**Issue by ammargitham** [Mr-Pine/AndroidUtilityLibraries#19]
Currently the panning stops once you lift your finger. For the panning to feel more natural, it would be great if there was an optional fling gesture support.

Found [this example code](https://developer.android.com/jetpack/compose/animation/advanced) on developer.android.com while searching around for a solution. But needs to be implemented in a `pointerInput` scope, so I think it has to be implemented inside the library.

**Comment by Mr-Pine** [comment]
Yeah, I'll look into it, can't say when though since Uni has just started up again but I'll try to get to it as soon as I can. Thanks for the example Code that'll be helpful and also gave me an Idea on how to fix the less than ideal double tap Syntax 👀
